jobs/nightly-rpm-builds.yml

```diff
@@ -13,7 +13,16 @@
         - project: gluster_build-rpms
           block: false
           predefined-parameters:
-            GERRIT_BRANCH=master
+            GERRIT_BRANCH=devel
```

Comment:
https://github.com/gluster/centosci/blob/master/jobs/build-rpms.yml#L22 does not list `devel` as valid branch.